doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -162,6 +162,7 @@ nRF Desktop
 -----------
 
 * Added an application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
+* Added application-specific Kconfig options (:kconfig:option:`CONFIG_DESKTOP_LOG` and :kconfig:option:`CONFIG_DESKTOP_SHELL`) to simplify the Debug configurations for the Logging and Shell subsystems.
```

Comment:
Please use the following syntax for nRF Desktop specific Kconfig options:
:ref:`CONFIG_DESKTOP_LOG<config_desktop_app_options>`

Reply:
fixed